Specs/Data/Tiles3D/instanced/instancedGltfEmbedded/tiles.json

```diff
@@ -0,0 +1 @@
+{"properties":{"id":{"minimum":0,"maximum":99},"height":{"minimum":0.7289306027814746,"maximum":97.26047022268176}},"geometricError":40,"root":{"box":[-1.319970476767974,0.6986942109000001,-1.3199295232320258,0.6987257891,0,15],"geometricError":0,"content":{"url":"instancedGltfEmbedded.i3dm"}}}
```

Comment:
For naming, I have been using `Cesium3DTiles` everywhere, which I do not like because it is so verbose.  This uses `Tiles3D`.  Perhaps that is better.  We should be consistent (file renames can be in a separate PR).  What do you suggest?